index.html

```diff
@@ -3,88 +3,108 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport"content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>paz</title>
     <link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
     <link href="dist/main.css" rel="stylesheet">
+    <link rel="stylesheet" href="node_modules/swiper/dist/css/swiper.css">
```

Comment:
You should move this `style` tag above yours (i.e. `<link href="dist/main.css" rel="stylesheet">`) so you can easily override styles from the swiper if you need to